src/views/product-tutorials-view/server.ts

```diff
@@ -51,12 +55,26 @@ export interface ProductPageData {
  *
  * @TODO add sidebar sort capability
  */
-export async function getProductTutorialsPageProps(
+export async function getProductTutorialsViewProps(
   productData: LearnProductData
-): Promise<{ props: ProductTutorialsPageProps }> {
+): Promise<{ props: ProductTutorialsViewProps }> {
   const productSlug = productData.slug
-  const { pageData, inlineCollections, inlineTutorials } =
-    await getProductPageContent(productSlug)
+  /**
+   * Get the raw page data
+   */
+  const {
+    pageData: rawPageData,
+    inlineCollections,
+    inlineTutorials,
+  } = await getProductPageContent(productSlug)
+  /**
+   * Add headings to raw page data, for use with the page's sidecar
+   */
+  const pageData = addHeadingSlugsToBlocks(rawPageData)
```

Comment:
🌟 This probably best captures the broad intent of the PR, being:
- Immediately after getting pageData from the learn client, add `headingSlug` values to it
- Use these `headingSlug` values for both the `<heading id={slug} />` element rendering, and the `Sidecar`